src/status_im2/contexts/profile/profiles/views_v2.cljs

```diff
@@ -0,0 +1,123 @@
+(ns status-im2.contexts.profile.profiles.views-v2
```

Comment:
I think this namespace is new anyway so there's no need for v2 👍

Reply:
Changed. It was in `status_im`. I just figured out it needs to be in `status_im2` at last minutes.